event-adapter/pkg/apiserver/installer/installer.go

```diff
@@ -250,13 +239,60 @@ func (a *EventsAPIInstaller) registerResourceHandlers(storage rest.Storage, ws *
 	addParams(namespacedRoute, namespacedParams)
 	ws.Route(namespacedRoute)
 
-	//REGISTER LIST ALL EVENT
+	//PATH namespaces/{namespace}/events/{eventName}
+
 	namespacedListPath := scope.ParamName() + "/{" + scope.ArgumentName() + "}/events"
 	namespacedListParams := []*restful.Parameter{
 		namespaceParam,
-		resourceParam,
 	}
 
+	doc = "event with the given namespace"
```

Comment:
Please be consistent with other docs. It should be in imperative form, like "list all events".

Reply:
Done